spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -66,14 +66,14 @@
         let!(:env_tag_mapping)         { FactoryBot.create(:tag_mapping_with_category, :label_name => "Category1") }
         let(:env_tag_mapping_category) { env_tag_mapping.tag.classification }
 
-        before do
-          collector.categories_by_id           = {category.id => category}
-          collector.tags_by_id                 = {tag.id => tag}
-          collector.tag_ids_by_attached_object = {"VirtualMachine" => {"vm-21" => [tag.id]}}
-        end
-
         it "saves vm labels" do
-          2.times { with_vcr { collector.refresh } }
+          2.times do
+            collector.categories_by_id           = {category.id => category}
+            collector.tags_by_id                 = {tag.id => tag}
+            collector.tag_ids_by_attached_object = {"VirtualMachine" => {"vm-21" => [tag.id]}}
```

Comment:
Now that we clear these after a refresh I had to move this inside the `2.times` block